inspire/base/static/js/search/search_results.js

```diff
@@ -44,8 +44,188 @@ require(['jquery', 'bootstrap'], function($) {
         $("#download" + data.recid).attr("href", "/formatter/download-latex/latex_us/" + data.recid)
       });
     });
+
   })
 
+  $('#formats-dropdown-menu > li').each(function(index) {
+    $(this).on("click", function() {
+      $("#dropdown-export").html($(this).text() + ' <span class="caret"></span>');
+    });
+  });
+
+  var sList = [];
```

Comment:
Create another variable with the `EXPORT_LIMIT` and use everywhere instead of the `5000`